.gitignore

```diff
@@ -34,7 +34,6 @@ __pycache__/
 /Code/GraphMol/SmilesParse/smiles.tab.hpp
 
 /rdkit/RDPaths.py
-/rdkit/Chem/inchi.py
```

Comment:
this should be in .gitignore. It is copied in by the build system if the RDKit is being built with InChI support.